pkg/utils/uuid_test.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewUUIDStr(t *testing.T) {
+	assert := assert.New(t)
+
+	output := NewUUIDStr()
+	outputType := reflect.TypeOf(output)
+	stringType := reflect.TypeOf("")
+
+	assert.IsType(stringType, outputType)
```

Comment:
I think instead of this you can check if the string is non-empty and that should be enough. I don't think checking the return type here is necessary.